main/inc/lib/sessionmanager.lib.php

```diff
@@ -5435,6 +5434,271 @@ public static function getCoursesListByCourseCoach($coachId)
     }
 
     /**
+     * Returns list of a few data from session (name, short description, start date, end date)
+     * And the next extra fields
+     * short_description, mode, human_text_duration, vacancies, brochure, target, schedule
+     * from Session category Id.
+     * @param int $categoryId
+     * @param string $target
+     * @return mixed
+     */
+    public static function getBriefSessionListAndExtraByCategory($categoryId, $target) {
```

Comment:
This is too specific to be included in 1.10.x. Either use an array as the third param of the method or return *all* extra fields, but you cannot give a specific hardcoded list.